src/world/area_dro/dro_01/npc_shop_owner.c

```diff
@@ -26,42 +26,46 @@ EvtScript N(EVS_NpcIdle_ShopOwner) = {
             EVT_CALL(SetNpcFlagBits, NPC_SELF, NPC_FLAG_IGNORE_PLAYER_COLLISION, TRUE)
             EVT_CALL(SetNpcPos, NPC_SELF, 0, 0, -180)
             EVT_EXEC_WAIT(N(EVS_OpenShopDoor))
-            EVT_WAIT(10)
+            EVT_WAIT(10 * DT)
             EVT_CALL(GetPlayerPos, LVarA, LVarB, LVarC)
             EVT_CALL(UseSettingsFrom, CAM_DEFAULT, LVarA, LVarB, LVarC)
             EVT_CALL(SetPanTarget, CAM_DEFAULT, 32, 0, -67)
             EVT_CALL(SetCamDistance, CAM_DEFAULT, -210)
             EVT_CALL(SetCamPitch, CAM_DEFAULT, 20, -12)
             EVT_CALL(SetCamSpeed, CAM_DEFAULT, EVT_FLOAT(90.0))
             EVT_CALL(PanToTarget, CAM_DEFAULT, 0, 1)
-            EVT_WAIT(20)
+            EVT_WAIT(20 * DT)
             EVT_CALL(UseSettingsFrom, CAM_DEFAULT, LVarA, LVarB, LVarC)
             EVT_CALL(SetCamDistance, CAM_DEFAULT, -350)
+#if VERSION_PAL
+            EVT_CALL(0x802CEEF0, 0, 0xF24A7E58) // Should be EVT_FLOAT(0.8 * DT) or 0.96, but that gets rounded 1 byte off
```

Comment:
Don't use raw function address if you can avoid it.
Try to work out the EVT_FLOAT macro here as well instead of the raw 0xF24A7E58.

Reply:
oh yeah I didn't see this - we shouldn't be merging any raw pointers into the codebase